lib/RexStan.php

```diff
@@ -166,9 +168,17 @@ public static function execCmd(string $cmd, &$lastError)
             $output = stream_get_contents($pipes[1]);
             fclose($pipes[1]);
 
-            $lastError = stream_get_contents($pipes[2]);
+            $stderrOutput = stream_get_contents($pipes[2]);
             fclose($pipes[2]);
 
+            $status = proc_get_status($process);
+            while (false !== $status && $status['running']) {
+                // sleep half a second
+                usleep(500000);
+                $status = proc_get_status($process);
+            }
+            $exitCode = false !== $status && $status['exitcode'];
```

Comment:
this check shouldn't be necessary, https://github.com/phpstan/phpstan/issues/8355